### DISCUSSION TITLE
Total Storage

### DISCUSSION THREAD

**Discussion by DKB0512** [rayanfer32/TgindexPro#24] [Q&A]
Can we somehow show in that index page that how much amount of storage that particular group is using, in terms of n number of GBs or something like that?

**Comment by rayanfer32** [comment]
Its unlimited ,but if you are curious about calculating the total size of the group then there isn't a straight foward api to get that.
We have to manually iterate over the messages

**Reply by pseudokawaii** [reply]
@rayanfer32 he's talking about the storage usage, not how much storage you have.

**Reply by rayanfer32** [reply]
I know , but it must be calculated iterating over all the messages i guess.

**Reply by pseudokawaii** [reply]
@rayanfer32 oh yes, and that is too taxing for the API. App'll get flood waited.

**Reply by DKB0512** [reply]
oh okay nvm, thanks for the reply tho :)
